pkg/utils/parse_arf_result.go

```diff
@@ -552,12 +569,32 @@ func getSafeText(nptr *xmlquery.Node, elem string) string {
 	return elemNode.InnerText()
 }
 
-func complianceCheckResultDescription(rule *xmlquery.Node) string {
+func complianceCheckResultDescription(rule *xmlquery.Node, valuesList map[string]string) (string, error) {
 	title := getSafeText(rule, "xccdf-1.2:title")
 	if title != "" {
 		title = title + "\n"
 	}
-	return title + getSafeText(rule, "xccdf-1.2:rationale")
+	description := rule.SelectElement("xccdf-1.2:description")
```

Comment:
looks good! Could we reduce code duplication by having a shared function where we would pass the element name and have the shared function be called by both `complianceCheckResultDescription` and `complianceCheckResultRationale`?